<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CounterForm</class>
 <widget class="QWidget" name="CounterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" columnstretch="8,0,2">
   <item row="1" column="1">
    <widget class="QSlider" name="zoomVerticalSlider">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="ImageGraphicsViewForm" name="imageGraphicsView">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="dragMode">
      <enum>QGraphicsView::NoDrag</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="loadPushButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Load Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fiToWindowPushButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Fit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="originalSizePushButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>1:1</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addGroupPushButton">
       <property name="text">
        <string>Rename Groups</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="mayflyCheckBox">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Mayfly</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chironomidaeCheckBox">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Chironomidae</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="QTableWidget" name="countTableWidget">
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <column/>
     <column/>
     <column/>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="savePushButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Save Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportPushButton">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Export Data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageGraphicsViewForm</class>
   <extends>QGraphicsView</extends>
   <header location="global">imagegraphicsviewform.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
